src/state/Maps/selectors.js

```diff
@@ -654,18 +689,26 @@ const getFinalLayerByDataSourceAndLayerState = createRecomputeSelector(
 		if (type === 'wmts') {
 			options.url = dataSourceOptions.url || dataSourceOptions.urls?.[0];
 		} else if (type === 'wms') {
-			const {url, params, configuration, ...rest} = dataSourceOptions;
+			let {url, params, configuration, ...rest} = dataSourceOptions;
 			const singleTile =
 				configuration && configuration.hasOwnProperty('singleTile')
 					? configuration.singleTile
 					: false;
 
 			const styles = rest.styles;
 
+			// if path is relative, add protocol & host from config
+			// TODO special config constants for mapserver?
+			if (url && url.charAt(0) !== 'h') {
+				const localConfig = AppSelectors.getCompleteLocalConfigurationObserver();
+				url = `${localConfig.apiBackendProtocol}://${localConfig.apiBackendHost}/${url}`;
```

Comment:
We should move it to the helper which identify relative URL and fix it to the absolute

Reply:
in future